static/app/views/dashboards/widgetBuilder/components/newWidgetBuilder.spec.tsx

```diff
@@ -69,6 +76,46 @@ describe('NewWidgetBuiler', function () {
     expect(await screen.findByRole('button', {name: '14D'})).toBeInTheDocument();
     expect(await screen.findByRole('button', {name: 'All Releases'})).toBeInTheDocument();
 
+    expect(await screen.findByPlaceholderText('Name')).toBeInTheDocument();
+    expect(await screen.findByTestId('add-description')).toBeInTheDocument();
+
     expect(await screen.findByText('TEST WIDGET')).toBeInTheDocument();
   });
+
+  it('edits name and description', async function () {
+    const mockNavigate = jest.fn();
+    mockUseNavigate.mockReturnValue(mockNavigate);
+
+    render(<DevWidgetBuilder isOpen onClose={onCloseMock} />, {
+      router,
+      organization,
+    });
+
+    await userEvent.type(await screen.findByPlaceholderText('Name'), 'some name');
+    // use this because navigate is called many times while the field is being updated
```

Comment:
QQ how many times is this being called? once for each character? I was hoping that the debounce would only actually trigger the navigate call once

Reply:
Also I think there's something to assert the last call:
https://jestjs.io/docs/expect#tohavebeenlastcalledwitharg1-arg2-

Can you see if we can use that?

Reply:
update: i lied the test seems to be taking 6 seconds on first run rippp

Reply:
https://github.com/getsentry/sentry/actions/runs/12122580769/job/33796139350?pr=81497#step:6:151

duration seems okay so far, otherwise it'd complain with a big red number next to it

Reply:
ah yes that was locally so it could be something with my environment